packages/pyo3_samples/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: pyo3_samples
+  version: 0.1.0
+source:
+  # url: https://github.com/hoodmane/pyo3_samples/tarball/main
+  # sha256: 5a402fc5323365cd2fce3379a7a619e6dba28083bf7eed1ec554820c1551dce2
+  path: src
+build:
+  sharedlibrary: true
+  script: |
+    source $HOME/.cargo/env
+    rustup update nightly
```

Comment:
Which features do you need nightly for, last stable release doesn't work?

Reply:
It's needed to build the standard library with `-Z build-std`. Ideally we could figure out how to build the standard library as a separate shared library. I think it's several megabytes big and we will eventually have more than one rust package if we get this working so it'd be better not to duplicate that anyways. But I don't know how to do this.